pdk/1.0/modules/ROOT/pages/policies-pdk-compile-policies.adoc

```diff
@@ -0,0 +1,41 @@
+= Compiling Custom Policies
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images
+
+You can compile the custom policy project after you complete all the steps in xref:policies-pdk-create-project.adoc[].
+
+You must compile the custom policy before you can deploy it to a Flex Gateway.
+
+After you compile your project, test your policy with the following tutorials:
+
+* xref:policies-pdk-debug-local.adoc[]
+* xref:policies-pdk-apply-policies.adoc[]
+
+== Before You Begin
+
+Ensure you have completed all the steps in xref:policies-pdk-create-project.adoc[].
+
+== Build Command
```

Comment:
Section titles in tutorials should be imperative, something like:
"Run the Build Command", or "Compile Using the Build Command".

Otherwise it reads like a reference doc, which I don't think this is.